playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -229,3 +229,56 @@
   tasks:
   - include: docker/upgrade.yml
     when: l_docker_upgrade is defined and l_docker_upgrade | bool and not openshift.common.is_atomic | bool
+
+- name: Drain and upgrade master nodes
```

Comment:
Is it possible to integrate this directly into the main play for masters above where we go through them 1 by 1 in serial? This is where the ELB hooks are and after discussions with ops I think we should aim to fully process each master in that loop, as much as possible, so when that system goes back into the ELB it's been fully updated.

As an example if this were a containerized cluster and we were using hooks to put into / out of an ELB, we would be back in the ELB but then later restart docker and bring down our services again.